devicetypes/smartthings/ezex-smart-occupancy-sensor.src/ezex-smart-occupancy-sensor.groovy

```diff
@@ -0,0 +1,106 @@
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations
+ *  under the License.
+ *  Author : Fen Mei / f.mei@samsung.com
+ *  Date : 2019-02-12
+ */
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
+import physicalgraph.zigbee.zcl.DataType
+metadata {
+    definition(name: "eZEX Smart Occupancy Sensor(AC Type)", namespace: "smartthings", author: "SmartThings", runLocally: false, mnmn: "SmartThings", vid: "SmartThings-smartthings-ZigBee_Dimmer_with_Motion_Sensor") {
+        capability "Motion Sensor"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Health Check"
+        capability "Sensor"
+       
+        fingerprint profileId: "0104", deviceId: "0107", inClusters: "0000, 0003, 0004, 0406", outClusters:"0006, 0019", manufacturer:"", model:"E280-KR0A0Z0-HA", deviceJoinName: "eZEX Smart Occupancy Sensor(AC Type)"
+    }
+    tiles(scale: 2) {
+        multiAttributeTile(name: "motion", type: "generic", width: 6, height: 4) {
+            tileAttribute("device.motion", key: "PRIMARY_CONTROL") {
+                attributeState "active", label: 'motion', icon: "st.motion.motion.active", backgroundColor: "#00A0DC"
+                attributeState "inactive", label: 'no motion', icon: "st.motion.motion.inactive", backgroundColor: "#cccccc"
+            }
+        }
+
+        standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat", width: 2, height: 2) {
+            state "default", action: "refresh.refresh", icon: "st.secondary.refresh"
+        }
+        main(["motion"])
+        details(["motion", "refresh"])
+    }
+}
+
+def installed(){
+    log.debug "installed"
+    return zigbee.readAttribute(0x0406, 0x0000)
+}
+
+def parse(String description) {
+    log.debug "description(): $description"
+    def map = zigbee.getEvent(description)
+    if (!map) {
+		if (description?.startsWith('read attr')) {
+            map = parseReportAttributeMessage(description)
+        } else {
+           log.debug "not handle"
+        }
+    }
+    log.debug "Parse returned $map"
+    def result = map ? createEvent(map) : [:]
+    if (description?.startsWith('enroll request')) {
+        List cmds = zigbee.enrollResponse()
+        log.debug "enroll response: ${cmds}"
+        result = cmds?.collect { new physicalgraph.device.HubAction(it) }
+    }
+    return result
+}
+
+private Map parseReportAttributeMessage(String description) {
+        Map descMap = (description - "read attr - ").split(",").inject([:]) { map, param ->
+                def nameAndValue = param.split(":")
+                map += [(nameAndValue[0].trim()):nameAndValue[1].trim()]
+        }
+        log.debug "Desc Map: $descMap"
+
+        Map resultMap = [:]
+        if (descMap.cluster == 0x0406 && descMap.attrId == 0x0000) {
```

Comment:
You have to use the fields `clusterInt` and `attrInt` when comparing against integer values.  The reason we want to use the integer values is that we've run into some issues where DTHs break because the capitalization of the hex changes or things like that, so using the integer values is safer.

Reply:
This device reported message have no clusterInt and attrInt fields, 
log:
Desc Map: [raw:D0440104060800001800, dni:D044, endpoint:01, cluster:0406, size:08, attrId:0000, encoding:18, value:00]
6eb7ce5d-18f4-4dca-92c4-28a49bbc2453  下午1:54:52: debug description(): read attr - raw: D0440104060800001800, dni: D044, endpoint: 01, cluster: 0406, size: 08, attrId: 0000, encoding: 18, value: 00

Reply:
When you call `parseDescriptionAsMap` with it will return the map that includes those fields.  So you can use them here.